### DISCUSSION TITLE
Ricepp headers in package?

### DISCUSSION THREAD

**Discussion by silentnoodlemaster** [mhx/dwarfs#195] [Q&A] [answered]
cmake creates these files, do I need to package them for ricepp support to work

     /usr/include/ricepp/bitstream_reader.h
     /usr/include/ricepp/bitstream_writer.h
     /usr/include/ricepp/byteswap.h
     /usr/include/ricepp/codec.h
     /usr/include/ricepp/codec_interface.h
     /usr/include/ricepp/detail/decode.h
     /usr/include/ricepp/detail/encode.h
     /usr/include/ricepp/ricepp.h
     /usr/lib64/cmake/ricepp/ricepp-config.cmake
     /usr/lib64/cmake/ricepp/ricepp-targets-release.cmake
     /usr/lib64/cmake/ricepp/ricepp-targets.cmake
     /usr/lib64/libricepp.a

**Comment by mhx** [comment, marked as answer]
Sorry, those shouldn't have been installed in the first place. This will be fixed in the next release.

In the meantime, no, you don't have to package them.

**Reply by mhx** [reply]
Should be fixed in v0.9.4.
